lib/dotcom_web/plugs/schedule_redirector.ex

```diff
@@ -0,0 +1,25 @@
+defmodule DotcomWeb.Plugs.ScheduleRedirector do
+  @moduledoc """
+  Catch-all for unsupported `/schedules/:route_id/*` URLs.
+  Redirects to the default `/schedules/:route_id` path.
+  """
+
+  @behaviour Plug
```

Comment:
🌤️ I like to put the behaviours after the include statements so they're closer to the implementation definitions.